examples/weekly-summary-email/steps/run/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM golang:1.18.2-alpine AS build
+WORKDIR /app
+COPY vendor vendor
+COPY . .
+ARG TARGETARCH
+ARG TARGETOS
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    GOFLAGS=-mod=vendor GOOS=$TARGETOS GOARCH=$TARGETARCH go build -o /go/bin/main cmd/main.go
```

Does this need to also include `mock.go`?

```suggestion
RUN --mount=type=cache,target=/root/.cache/go-build \
    GOFLAGS=-mod=vendor GOOS=$TARGETOS GOARCH=$TARGETARCH go build -o /go/bin/main cmd/main.go cmd/mock.go
```

For my benefit, is it best practice to list the exact files needed here or should we use `cmd/*.go`?

---

aha this should definitely be cmd/*.go, thanks!